clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelPropertiesTableRow.tsx

```diff
@@ -0,0 +1,187 @@
+import * as React from 'react';
+import { ActionsColumn, Td, Tr } from '@patternfly/react-table';
+import {
+  ActionList,
+  ActionListItem,
+  Button,
+  ExpandableSection,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  TextInput,
+} from '@patternfly/react-core';
+import { CheckIcon, TimesIcon } from '@patternfly/react-icons';
+import { KeyValuePair } from '~/types';
+import { EitherNotBoth } from '~/typeHelpers';
+
+type ModelPropertiesTableRowProps = {
+  allExistingKeys: string[];
+  setIsEditing: (isEditing: boolean) => void;
+  isSavingEdits: boolean;
+  setIsSavingEdits: (isSaving: boolean) => void;
+  saveEditedProperty: (oldKey: string, newPair: KeyValuePair) => Promise<unknown>;
+} & EitherNotBoth<
+  { isAddRow: true },
+  {
+    isEditing: boolean;
+    keyValuePair: KeyValuePair;
+    deleteProperty: (key: string) => Promise<unknown>;
+  }
+>;
+
+const ModelPropertiesTableRow: React.FC<ModelPropertiesTableRowProps> = ({
+  isAddRow,
+  isEditing = isAddRow,
+  keyValuePair = { key: '', value: '' },
+  deleteProperty = () => Promise.resolve(),
+  allExistingKeys,
```

Comment:
Take a look at the latest version, it seems they've added a new prop to detect if the model is archived: https://github.com/opendatahub-io/odh-dashboard/pull/3206

Reply:
That is addressed already in the PR that will follow this one